collector/docker/kindling-collector-config.yml

```diff
@@ -118,6 +122,10 @@ processors:
         - kind: sum
       response_io:
         - kind: sum
+      kindling_pagefault_major_total:
+        - kind: last
+      kindling_pagefault_minor_total:
+        - kind: last
```

Comment:
If the type of the two metrics is "last", then the metric name should not be ended with "_total" which is designed for the "counter" type.